app/routes/admin/groups/components/GroupMembers.js

```diff
@@ -95,6 +95,7 @@ function loadData({ query, match: { params }, location }, dispatch) {
 
 function mapStateToProps(state, props) {
   const { pathname, search } = state.router.location;
+  // eslint-disable-next-line no-restricted-globals
```

Comment:
What about fixing this instead of ignoring it, since the warning is legit?

Reply:
You're right. I fixed this with `window.location` instead of `location` and `ConfirmModalWithParent` instead of `confirm()`